spec/pluck_to_tstruct_spec.rb

```diff
@@ -43,6 +67,22 @@ def eql?(other)
     end
   end
 
+  class WizardWithWandT < T::Struct
+    const :name, String
+    const :house, String
+    const :wand_wood_type, String
+    const :wand_broken, Integer
```

Comment:
I'm not sure why but when we pluck for this in the test it returns an `Integer` representation of the `boolean`. When testing my changes using our DB `boolean` columns return as `true` or `false`. Not sure if this is a problem just wanted to point it out

Reply:
seems like this is causing issues in CI as well https://travis-ci.com/github/chanzuckerberg/sorbet-rails/jobs/343825939

Any ideas?

Reply:
without investigating much, i'm guessing it's because the tests are running on sqlite.

https://www.sqlite.org/datatype3.html#:~:text=Boolean%20Datatype,)%20and%201%20(true).

Reply:
if i were you i'd be lazy and just not use a bool field at all for this test.

Reply:
+1 Alex's comment